src/v2/Apps/Example/README.md

```diff
@@ -10,3 +10,9 @@ It includes:
 - [ ] Testing
 - [ ] Analytics
 - [ ] Mention of adding tests to integrity
+
+## Launch checklist
+
+- [ ] Ensure meta tags are fleshed out and correct (title, description, image, canonical)
+- [ ] If migrating a legacy app, ensure any redirects or route aliases are ported over
+- [ ] Add the new page to [the list of analytics exclusions](https://github.com/artsy/force/blob/master/src/desktop/assets/analytics.ts)
```

Comment:
@damassi anything obvious I'm missing?

Reply:
We should probably add a line about analytics sign off with Data team